lxd/instance/drivers/driver_lxc.go

```diff
@@ -6800,7 +6876,21 @@ func (d *lxc) FileSFTP() (*sftp.Client, error) {
 
 // stopForkFile attempts to send SIGINT to forkfile then waits for it to exit.
 func (d *lxc) stopForkfile() {
```

Comment:
I think this function should take a a `force bool` argument to trigger an immediate shutdown, rather than sending it the SIGINT and waiting for it to cleanly shutdown.